doc/man/cockpit.conf.xml

```diff
@@ -196,6 +196,19 @@ ProtocolHeader = X-Forwarded-Proto
     </variablelist>
   </refsect1>
 
+  <refsect1 id="cockpit-conf-session">
+    <title>Session</title>
+    <variablelist>
+      <varlistentry>
+        <term><option>Banner</option></term>
+        <listitem>
+          <para>The contents of the specified file are shown on login page before authentication is allowed.
```

Comment:
I'd drop the "before authentication is allowed", as it's somewhat redundant and also confusing -- you can authenticate with kerberos or a smart card without seeing the login page. Also, "on *the* login page".

Also, what about giving some hint for re-using a standard API: "specified file (commonly `/etc/issue`)"